openshift/auth.config.yaml

```diff
@@ -31,6 +29,8 @@ objects:
   data:
     service-account-secrets: >-
       ewogICJhY2NvdW50cyI6IFsKICAgICAgewogICAgICAgICAgIm5hbWUiOiJmYWJyaWM4LXdpdCIsCiAgICAgICAgICAiaWQiOiI1ZGVjNWZkYi0wOWUzLTQ0NTMtYjczZi01YzgyODgzMmIyOGUiLAogICAgICAgICAgInNlY3JldHMiOlsid2l0c2VjcmV0Il0KICAgICAgfSwKICAgICAgewogICAgICAgICAgIm5hbWUiOiJmYWJyaWM4LXRlbmFudCIsCiAgICAgICAgICAiaWQiOiJjMjExZjFiZC0xN2E3LTRmOGMtOWY4MC0wOTE3ZDE2Nzg4OWQiLAogICAgICAgICAgInNlY3JldHMiOlsidGVuYW50c2VjcmV0T2xkIiwgInRlbmFudHNlY3JldE5ldyJdCiAgICAgIH0KICBdCn0=  
+    oso-clusters: >-
```

Comment:
As per my previous comment, this secret value is defined under the fabric8-auth-service-accounts secret.  I would be inclined to put this value in a separate secret configuration, like so:

```
- apiVersion: v1
  kind: Secret
  metadata:
    name: fabric8-auth-oso-clusters
  type: Opaque
  data:
    oso-clusters: >-
      ewogICJhY2NvdW50cyI6IFsKICAgICAgewogICAgICAgICAgIm5hbWUiOiJmYWJyaWM4LXdpdCIsCiAgICAgICAgICAiaWQiOiI1ZGVjNWZkYi0wOWUzLTQ0NTMtYjczZi01YzgyODgzMmIyOGUiLAogICAgICAgICAgInNlY3JldHMiOlsid2l0c2VjcmV0Il0KICAgICAgfSwKICAgICAgewogICAgICAgICAgIm5hbWUiOiJmYWJyaWM4LXRlbmFudCIsCiAgICAgICAgICAiaWQiOiJjMjExZjFiZC0xN2E3LTRmOGMtOWY4MC0wOTE3ZDE2Nzg4OWQiLAogICAgICAgICAgInNlY3JldHMiOlsidGVuYW50c2VjcmV0T2xkIiwgInRlbmFudHNlY3JldE5ldyJdCiAgICAgIH0KICBdCn0=

```

Reply:
Good catch. Maybe we should move all the secret values to the same “auth” secret instead? WDYT? Including all values that we mount as env vars and config files. Or alternatively one secret for env var values and one (auth-config-files?) for service accounts and oso clusters.
WDYT?